nextflow.config

```diff
@@ -8,6 +8,7 @@ params {
   max_time = 240.h
 
   ref = 'https://github.com/czbiohub/test-datasets/raw/msspe/reference/MN908947.3.fa'
+  ref_host = 'https://github.com/czbiohub/test-datasets/raw/olgabot/subset-chrom-I-gzip/reference/genome.fa'
```

Comment:
It looks like this fasta only has one chromosome. Please upload the full human genome somewhere accessible and use that as the default `ref_host`. Also, please rerun the benchmark using the full human genome.

Reply:
Oh woops. Yeah I ran it with the full human genome earlier but only used chrm1 for the testing since the full human genome wasn't easily accessible anywhere for this. Do you know a good place to host this? I could use a public s3 bucket if there isn't a better options of an existing source you are aware of.

Reply:
I realized I'm just assuming that hosting it on github the way the existing test data is stored wouldn't work due to the size but it might be fine. I'll try that first.